.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Support Request
+about: Support request or question relating to cloud-prepare
+labels: support
+
+---
+
+<!--
+GitHub may not the right place for support requests.
```

Comment:
```suggestion
GitHub may not be the right place for support requests.
```

Reply:
That needs to be updated across all other repos as well, good catch :)